rootfs/etc/confd/templates/builder

```diff
@@ -64,17 +64,30 @@ TMP_DIR=$(mktemp -d -p $BUILD_DIR)
 
 cd $REPO_DIR
 # use Procfile if provided, otherwise try default process types from ./release
+
+git archive --format=tar.gz ${GIT_SHA} > ${APP_NAME}.tar.gz
```

Comment:
I did a ls of files inside repo direcotry and it didn't show up any files related to example-go, but when I did archive and untar the files are there. You can play with smothiki/builder:3.1.0 to observe this behavior. reason for doing a git archive and then untar the files in a temp direcotry.
This might be issue with previous builder as well but its passing because we have procfile check from generated slug.